test/functional/apps/getting_started/_shakespeare.js

```diff
@@ -68,7 +68,7 @@ export default function ({ getService, getPageObjects }) {
         const data = await PageObjects.visualize.getBarChartData('Count');
         log.debug('data=' + data);
         log.debug('data.length=' + data.length);
-        expect(data).to.eql(expectedChartValues);
+        expect(data[0] - expectedChartValues[0]).to.be.lessThan(5);
```

Comment:
I'm OK with this, but just as an FYI, in visualize_page getLineChartData we round the returned value to 6 digits here;

https://github.com/elastic/kibana/blob/master/test/functional/page_objects/visualize_page.js#L886

In this case I guess we'd have to round to 5 digits, but even that is still pretty good for checking the height of a bar on a chart.  But if you change that you'd have to change the expected result in 14 places.  So your change is fine.

Reply:
Good point, Lee. I see my change as a temporary fix and think that moving at least some of these tests to visual regression suite is a better option.